web/src/title/NewProjectMode.svelte

```diff
@@ -44,7 +45,11 @@
       );
 
       const projectID = $projectStorage.createProject($backend.toSavefile());
-      await loadProject(projectID);
+      // Calling loadProject would immediately re-fetch OSM data from Overpass
```

Comment:
The first thing I tried was making `OverpassSelector` just hand us the boundary GJ and not actually load anything from Overpass itself. Then we could create an empty project and immediately load it, instead of having this second slightly weird code path. That approach would be cleaner, but I'm too lazy currently to go fix OverpassSelector and figure out if that kind of change also makes sense in other projects using it :)